### ISSUE TITLE
Install Help

### ISSUE THREAD

**Issue by dregster77** [mtgeekman/Home_Assistant_NeoSmartBlinds#51]
I am new to home assistant and I am trying to get this setup for my Smart blinds controller but I installed it through hacs and then I am reading the instructions and the first one says to add it to the configuration. yaml file when it says that do I need to make one in the neo Smart blinds folder or is it saying to add that to the base configuration.yaml file?

**Comment by keithr59** [comment]
You need to add it to the base configuration.yaml file.

**Comment by dregster77** [comment]
awesome thanks. also is there any way for ha to know the status of up or down for the blinds?

**Comment by keithr59** [comment]
When the integration is initialised HA doesn't know what position a blind is in so it sets its state to Unknown. 

I have an automation that is triggered when HA starts and, depending on the time of day, will close/open the blinds to synchronise their state.

**Comment by dregster77** [comment]
can ha have automations triggered when it restarts or boots up?

**Comment by mtgeekman** [comment]
Yep, when you create a new automation and add a trigger, click the "Other Triggers" item at the bottom, its the second in the list that comes up.

**Comment by dregster77** [comment]
O wow that is awesome thanks!
